lib/Service/RoomService.php

```diff
@@ -362,6 +365,42 @@ public function setAvatar(Room $room, string $avatar): bool {
 		return true;
 	}
 
+	public function startRecording(Room $room, $status): bool {
+		$availableRecordingTypes = [self::RECORDING_VIDEO, self::RECORDING_AUDIO];
+		if (!in_array($status, $availableRecordingTypes)) {
+			return false;
```

Comment:
I would recommend to follow the new pattern and throw with a keyword here instead of returning a boolean false.
Check the breakout service